upload/tokenless/github_actions.py

```diff
@@ -52,9 +52,14 @@ def get_build(self):
                 retry_after = int(e.retry_after)
             else:
                 retry_after = None
+            time_to_available_str = ""
+            if retry_after is not None:
+                time_to_available_str = (
+                    f" Expected time to availability: {retry_after}s."
+                )
             raise exceptions.Throttled(
-                wait=min(10, retry_after) if type(retry_after) == int else retry_after,
-                detail="Rate limit reached. Please upload with the Codecov repository upload token to resolve issue.",
+                wait=None,
+                detail=f"Rate limit reached. Please upload with the Codecov repository upload token to resolve issue.{time_to_available_str}",
```

Comment:
[todo] taking a look at this "Please upload with the Codecov repository upload token to resolve issue" message, I think it's inaccurate, but I'm not sure how else to phrase this. Perhaps it would make sense to add a page to the docs that explain _why_ using the repo upload token may help because it means we can decide to use a github installation if it exists for that user.

Reply:
@joseph-sentry oh that's a really good point, let me noodle on this for a bit...

Reply:
https://github.com/codecov/feedback/issues/364